sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobAsyncClientBase.java

```diff
@@ -258,7 +258,7 @@ protected BlobAsyncClientBase(HttpPipeline pipeline, String url, BlobServiceVers
      *
      * @return the encryption scope used for encryption.
      */
-    public String getEncryptionScope() {
+    protected String getEncryptionScope() {
```

Comment:
Why are we changing this back to protected? Why was this changed to public in the first place?

Reply:
It was changed to public in the first place for testing purposes and to be more consistent with the sync client as the sync client has this method as public. However, in the API review, this change was determined to be breaking so it is being switched back to protected and the test (getEncryptionScopeClient() in CPKNAsyncTests.java) has been rewritten.